packages/cozy-client/src/models/paper.js

```diff
@@ -0,0 +1,97 @@
+import add from 'date-fns/add'
+import sub from 'date-fns/sub'
+
+import { IOCozyFile } from '../types'
```

Comment:
Since you use it only for the typing, you should remove this JS import and replace it by: 
```
@typedef {import("../types").IOCozyFile} IOCozyFile
```

Reply:
Done. Did the same for `Qualification`